platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -547,6 +550,8 @@ object Capture {
 
             default.set(LoggerState.Started(loggerImpl))
 
+            fatalIssueReporter?.initBuiltInMode(appContext, clientAttributes, loggerImpl)
```

Comment:
why are we handling all the construction and initialization of the fatalIssueReporter outside of LoogerImpl? Seems like it deviates from the current pattern. Seems like it could all be done inside the Logger instantiation like everything else